code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -35,21 +35,11 @@
 	//Temperature effect
 	var/minbodytemp = 250
 	var/maxbodytemp = 350
-	var/heat_damage_per_tick = 3	//amount of damage applied if animal's body temperature is higher than maxbodytemp

```

Comment:
I think people would prefer these instead of hard coded proc rewrites that you can't modify at runtime